### ISSUE TITLE
Fails to compile on latest nightly

### ISSUE THREAD

**Issue by theoparis** [immunant/c2rust#1098]
See https://github.com/time-rs/time/issues/681, it seems like some dependencies need to be updated since it isn't used directly...

Adding the following to Cargo.toml in this repository seems to fix it for the time being.
```
[patch.crates-io]
time = { git = "https://github.com/time-rs/time" }
```

Update: I think the correct fix would actually be updating Cargo.lock and the time-macros crate.

**Comment by tommythorn** [comment]
With the risk of stating the obvious, this is also causing `cargo install c2rust` to fail.

**Comment by chrysn** [comment]
Note that c2rust compiled just fine also on stable Rust. So as theoparis mentioned in the update, unlocking time-rs-macros sounds like the way to go. We need time 0.3.35 to fix the build issue, which needs Rust 1.67, which is more recent than the currently locked nightly.

**Comment by kkysen** [comment]
* Fixed by #1115.
* Will be released after #1116.
